apps/web/src/modules/job/components/JobCardFooter/index.tsx

```diff
@@ -11,12 +11,27 @@ import { FooterRow, SameDiv } from './styled'
 import { FooterProps } from './types'
 
 const JobCardFooter = (prop: FooterProps) => {
-  const { actorCount, wage, status, dueDate, gender, jobId, isApplied } = prop
+  const {
+    actorCount,
+    wage,
+    status,
+    dueDate,
+    gender,
+    jobId,
+    isApplied,
+    appliedStatus,
+    openModal,
+  } = prop
   const { user } = useUser()
 
   const apply: React.MouseEventHandler<HTMLButtonElement> = (e) => {
     e.stopPropagation()
   }
+  const cancel: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+    e.stopPropagation()
+    e.preventDefault()
+    openModal!()
```

Comment:
(Optional)

To prevent accidentally error from `undefined`

```suggestion
    openModal?.()
```